src/Plugin.php

```diff
@@ -73,18 +73,19 @@ public static function init() {
       // Saves custom fields for simple products.
       add_action('woocommerce_process_product_meta', __NAMESPACE__ . '\WooCommerce::woocommerce_process_product_meta');
 
+      // Adds data-srcset attributes to image links to make them reponsive in
+      // lightGallery.
+      // @see https://github.com/netzstrategen/wordpress-gallerya/pull/11#issuecomment-355664739
```

Comment:
```suggestion
      // @todo Make lightGallery properly respect srcset & sizes in JavaScript
      //   instead of duplicating that information in HTML.
      // @see https://github.com/netzstrategen/wordpress-gallerya/pull/11#issuecomment-355664739
```